index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portifolio mobile</title>
+  <link rel="stylesheet" href="index.css" />
+</head>
+
+<body>
+  <div class="font">
+    <section class="first-section">
+      <img src="./images/Snapshoot-icon.jpg" alt="Snapshoot" />
+
+      <h3 class="project_name">
```

Comment:
- [ ] You should implement the `works` and `About` sections below the sections you worked on them in the previous project (header and introduction section). You are building your Portfolio step by step. Kindly add the header and headline section above the `works` section.